CoreUtilities/src/au/gov/asd/tac/constellation/utilities/icon/IconData.java

```diff
@@ -171,6 +172,31 @@ private static BufferedImage scaleImage(final BufferedImage image, final int siz
         }
     }
 
+    @Override
+    public int hashCode() {
+        int hash = 7;
+        hash = 79 * hash + Arrays.hashCode(this.data);
+        return hash;
```

Comment:
if you're able to merge this all into one statement, that would be great (really don't understand why the auto-generated code expands it out like this)

Reply:
Updated